src/CMakeLists.txt

```diff
@@ -397,12 +397,14 @@ set(common_SRCS
 	genericobject.cpp
 	gettext.cpp
 	httpfetch.cpp
+	hud.cpp
 	inventory.cpp
 	inventorymanager.cpp
 	itemdef.cpp
 	itemstackmetadata.cpp
 	light.cpp
 	log.cpp
+	main.cpp
```

Comment:
i don't see why this has been moved here ? to prevent dup ?

Reply:
yes